docs/consensus/consensus/plotting.md

```diff
@@ -82,8 +82,8 @@ When a sector reaches its expiry point, the block proposer challenge solutions c
 
 In a plot spanning multiple gigabytes, the sectors will be updated randomly, one at a time, so replotting is amortized over a long period. There is never a moment when a farmer needs to erase and re-create their whole plot and miss out on challenges. The plot refreshing will be practically invisible to the farmer and allow their uninterrupted participation in consensus.
 
-The bigger the chain grows and the longer the farmer participates in the network, the less frequent the replotting on their disks will be.
-Assuming a 1 TB SSD plot, a farmer who joined at genesis will on average need to replot 7.7 TB of data by the time the chain history reaches 1TB in size, 10.5 TB of data by the time history reaches 10 TiB and 12.3 TB by the time history reaches 50 TiB. Given a common TBW (Total Bytes Written) for consumer grade 1 TB SSDs of 300-600 TB, Subspace farming requires below 3% of the SSD's total endurance to farm over several years (for comparison, Ethereum's full chain data size is ~16 TiB as of October 2023 via [Etherscan](https://etherscan.io/chartsync/chainarchive)).
+The bigger the chain grows and the longer the farmer participates in the network, the less frequent the replotting on their disks will be. After genesis Subspace will seed the network with 20 GiB of history, which means that farmers who join after the seeding will start plotting with initial history size of 160 segments.
```

Comment:
This is just an example for 3h, it will likely be very different for mainnet

Reply:
Sure, I will update for mainnet if needed